.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Feature Request
+about: Make suggestions on how TM:PE could improve.
+title: ''
+labels: 'feature'
```

Comment:
add `triage` label too?

(the triage label means that we can quickly list any issues that none of the team have looked at; after looking at an issue, we can remove the triage label)